### ISSUE TITLE
Feature Request: Images Filtering

### ISSUE THREAD

**Issue by ekianjo** [stolowski/QComicBook#11]
When one gets comics from online sources, sometimes scans are not of the best quality and suffer from compression artifacts and/or poor resolution. There are actually filters out there to help improve the quality of such images significantly - Waifu2x is one (but not easy I think), while other filters such as NNEDI3 (http://bengal.missouri.edu/~kes25c/) in Avisynth that would be great if they could be integrated / used together with qComicbook.

**Comment by stolowski** [comment]
Absolutely. I was experimenting with some graphics filtering in qcomicbook a long time ago (long enough to completely forget where I left off...). It would definately be a great enhancement. Thanks for pointers for existing algorightms, will keep them in mind when I find time to work on new release.

**Comment by ekianjo** [comment]
Thanks - sounds good. One thing that I'm not sure : whether it makes sense to apply those filters in "real time" before displaying an image, or if it's a better approach to apply it to the whole folder in the background. Depending on how computationally intensive those filters are, you want to consider either way.
